crates/astria-conductor/local.env.example

```diff
@@ -32,3 +32,19 @@ ASTRIA_CONDUCTOR_SEQUENCER_URL="ws://127.0.0.1:26657/websocket"
 
 # the sequencer block height that the rollup's first block was in
 ASTRIA_CONDUCTOR_INITIAL_SEQUENCER_BLOCK_HEIGHT=1
+
+# set to true to enable op-stack deposit derivations
+ASTRIA_CONDUCTOR_ENABLE_OPTIMISM=false
+
+# Websocket URL of Ethereum L1 node.
+# Only used if `enable_optimism` is true.
+ASTRIA_CONDUCTOR_ETHEREUM_L1_URL="ws://127.0.0.1:8545",
```

Comment:
Should this be prefixed with `OPTIMISM_ETHEREUM`? Dunno, no hard opinion on this,

Reply:
i'd rather leave as-is - i think `OPTIMISM_ETHEREUM` might get confusing?